include/vrv/note.h

```diff
@@ -190,6 +190,7 @@ class Note : public LayerElement,
     int GetChromaticAlteration();
 
     TransPitch GetTransPitch();
+    void UpdateFromTransPitch(TransPitch tp);
```

Comment:
Since they will be called from `Note::Transpose` they should be made `private`

Reply:
Oh, cool! (I guess I didn't notice that earlier).

Updated in 52cef660